bebop_driver/src/bebop.cpp

```diff
@@ -549,6 +549,20 @@ void Bebop::Move(const double &roll, const double &pitch, const double &gaz_spee
   }
 }
 
+
+void Bebop::MoveBy(const float& dX, const float& dY, const float& dZ, const float& dPsi)
```

Comment:
Please change `float` to `double` in the signature since the twist message fields (that you pass here) are all doubles.